src/open_inwoner/components/templates/components/Card/ProductCard.html

```diff
@@ -0,0 +1,31 @@
+{% load card_tags icon_tags link_tags button_tags i18n grid_tags thumbnail %}
+
+{# template for product cards with images #}
+{% render_card href=url grid=image title=title %}
+    {% if image %}
+        <div class="card__image-container">
+            <img class="card__image" src="{{ image|thumbnail_url:'card-image' }}" alt="{{ image.default_alt_text }}"/>
+        </div>
+        <div class="card__content">
+                <h2 class="h2"><span class="link link__text">{{ title }}</span></h2>
+            {% else %}
+                <div class="card__content">
+                <h2 class="h2"><span class="link link__text">{{ title }}</span></h2>
+            {% endif %}
```

Comment:
The indentation is a bit off here and makes it difficult to see the html/template tag nesting.